packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -278,12 +278,14 @@ describe('filterAuditsAndGroupsByOnlyOptions to be used in plugin config', () =>
       );
 
     expect(filteredAudits).toHaveLength(1);
```

Comment:
🧪 Redundancy: The `toHaveLength` assertions are no longer necessary when `toStrictEqual` is used on an array. Applicable in all affected test cases.

Reply:
Thx! Removed them on multiple places